sympy/polys/domains/old_fractionfield.py

```diff
@@ -101,9 +101,9 @@ def from_GlobalPolynomialRing(K1, a, K0):
         """Convert a ``DMF`` object to ``dtype``. """
         if K1.gens == K0.gens:
             if K1.dom == K0.dom:
-                return K1(a.rep)
+                return K1(a.to_list())
             else:
-                return K1(a.convert(K1.dom).rep)
+                return K1(a.convert(K1.dom).to_list())
```

Comment:
If `DMF` was based directly on `DMP` then we could just pass this through without calling `.to_list()`.